pom.xml

```diff
@@ -88,8 +88,13 @@
 		<dependency>
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-xml</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>commons-logging</groupId>
+					<artifactId>commons-logging</artifactId>
+				</exclusion>
```

Comment:
OK. That's enough. Thank you.
Although I remember as recently I was forced to exclude all `slf4j` transitive artifacts for Spring Kafka since one of PRs had used Slf4J directly. Removing it from `compile` makes us happy just with `commons-logging` available.
The Slf4J has been moved to the `testRuntime` to observe the logs anyway.

IMO that might be overhead here to track all the Slf4J for exclusion. :smile: